mixer/adapter/stackdriver/metric/metric.go

```diff
@@ -266,6 +268,7 @@ func (h *handler) HandleMetric(_ context.Context, vals []*metric.Instance) error
 
 func (h *handler) Close() error {
 	h.ticker.Stop()
+	h.quit <- 1
```

Comment:
why not just `close(h.quit)` ?  calling `Close()` twice should be against the contract, right?

Reply:
Done.